docs/profiles.md

```diff
@@ -301,3 +304,35 @@ The following command runs the tests and applies the secrets specified in `profi
     $ inspec exec examples/profile-attribute --attrs examples/profile-attribute.yml
 
 See the full example in the InSpec open source repository: https://github.com/chef/inspec/tree/master/examples/profile-attribute
+
+# Profile files
+
+An InSpec profile may contain additional files that can be accessed during tests. This coveres use-cases where e.g. a list of ports is provided to be tested.
```

Comment:
This feels like a semi-bad example of why you'd want to use a file because something like a port list is easy-enough to put in the control itself.

Maybe something like "This allows a profile to include a file that contains many different fields and values that could be used through the profile in across mulitple control files without including that data in each control file."